runtime/doc/api.txt

```diff
@@ -2763,6 +2763,10 @@ nvim_buf_set_extmark({buffer}, {ns_id}, {line}, {col}, {opts})
                     When a character is supplied it is used as |:syn-cchar|.
                     "hl_group" is used as highlight for the cchar if provided,
                     otherwise it defaults to |hl-Conceal|.
+                  • conceal_line: string which should be either empty or a
+                    single character. When empty, lines in the range are not
+                    drawn at all; the next unconcealed line is drawn instead.
+                    Supplying a character is currently unimplemented.
```

Comment:
```suggestion
                  • conceal_line: string (empty or a
                    single character). When provided, lines in the range are not
                    drawn at all; the next unconcealed line is drawn instead.
```
(Assuming that a non-empty string is simply ignored at the moment, meaning the behavior is the same.)

Reply:
> No objections, but I copied the first sentence from conceal verbatim.

That's fine then.

> If we keep that, your suggestion boils down to removing the mention of it being unimplemented?
Currently I also error when a single character is provided, could make it work like you say.

Yes, that was my suggestion.